src/js/common/components/AlertBox.jsx

```diff
@@ -33,7 +33,7 @@ class AlertBox extends React.Component {
 
   render() {
     if (!this.props.isVisible) {
-      return <div aria-live="assertive"/>;
+      return null;
```

Comment:
Since this isn't really in the scope of these set of changes, I'll just revert it for now. I think having a wrapper `div` with `aria-live` specifically for an alerts section in the header of an app would be a good solution, but that should be a separate PR.